test/extensions/filters/http/compressor/compressor_filter_speed_test.cc

```diff
@@ -520,6 +593,120 @@ BENCHMARK(compressChunks1024WithZstd)
     ->UseManualTime()
     ->Unit(benchmark::kMillisecond);
 
+static std::vector<CompressionParams> brotli_compression_params = {
```

Comment:
Is there a better way to generate it?

Reply:
I think it's more intuitive to list all params used and also convient to modify each param.

Reply:
no non-pod statics. You can use a std::array or C array here and specify it as constexpr. I'm not sure if your tuple implementation would work for CompressionParams in that case; you might need to make it a struct (which IMO would be more readable anyway :)

Reply:
OK, I have changed tuple to struct, but find that `constexpr` can't work here, so I change it to `const`. Thanks!